packages/bookshelf-transaction-events/lib/bookshelf-transaction-events.js

```diff
@@ -10,14 +10,14 @@ module.exports = function (bookshelf) {
             const orig2 = t.commit;
             const orig3 = t.rollback;
 
-            t.commit = function () {
+            t.commit = async function () {
+                await orig2.apply(t, arguments);
```

Comment:
Do we end up losing the return value of the original `.commit` etc by doing this? I can't find if there is one but it might be good to keep the same functionality as before? 🤔

Reply:
agreed 👍